Sources/SwiftDocC/Converter/TopicRenderReferenceEncoder.swift

```diff
@@ -38,10 +38,8 @@ enum TopicRenderReferenceEncoder {
         // The keys of the node render references
         var referenceIndexes = Array(references.keys)
         
-        // Sort the keys if `.sortedKeys` is set on the encoder.
-        if encoder.outputFormatting.contains(.sortedKeys) {
-            referenceIndexes.sort()
-        }
+        // Sort the render references so that RenderJSON output is stable and deterministic
+        referenceIndexes.sort()
```

Comment:
I don't think it makes sense to gate this sorting on the encoder settings since we sort the rest of the RenderJSON output regardless of that setting.

However, if this causes a performance impact in CI testing, I'll revert this change and we can explore this in a future PR.